fxa-oauth-server/test/api.js

```diff
@@ -2614,6 +2614,19 @@ describe('/v1', function() {
             assert.equal(Object.keys(res.result).length, 0, 'no scoped keys');
           });
       });
+
+      it('correctly verifies stale authAt', () => {
```

Comment:
nit: the name of this test seems confusing to me, suggesting that it verifies an authentication that it actually stale.  Something like "correctly handles authAt timestamp for newly-created accounts" might prevent future confusion.

Reply:
radical!